Backpacks.cs

```diff
@@ -651,6 +661,37 @@ public void Open(BasePlayer looter)
                     return;
                 }
 
+                if (looter.userID == OwnerId)
+                {
+                    // Check for items beyond the backpack's capacity
+                    var allowedCapacity = GetCapacity();
+
+                    var extraItems = _itemContainer.itemList
+                        .OrderBy(item => item.position)
+                        .Where(item => item.position >= allowedCapacity)
+                        .ToList();
```

Comment:
I think this doesn't necessarily need to be a list, please remove it unless you disagree.